spec/acceptance/splunk_spec.rb

```diff
@@ -5,6 +5,8 @@
     # Using puppet_apply as a helper
     it 'works idempotently with no errors' do
       pp = <<-EOS
+      class { '::splunk::params':
+      }
```

Comment:
I don't think you need to invoke `splunk::params` explicitly at all, do you?

Reply:
I agree :).  Let me see if I can remove this without things going haywire

Reply:
Done.